build.py

```diff
@@ -264,6 +264,8 @@ def core_cmake_args(components, backends, install_dir):
 
     if platform.system() == 'Windows':
         cargs.append('c:/workspace/build')
+    if FLAGS.jetson_build:
+        cargs.append(FLAGS.cmake_dir)
```

Comment:
--cmake-dir is a good addition, but use it to get rid of all these hardwired paths. The in CI find our other uses of build.py and pass in the appropriate value.

Reply:
Yes, will do!